package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+
+           # hbonds linking the selection 1 and selection 2 to the bridgeing water 2
+           [ # hbond 1 from selection 1 to the bridging water 2
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           [ # hbond 1 from selection 2 to the bridging water 2
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           ....
+        ],
+        [ # frame 2
+          [ ... ], [ ... ], ...
+        ],
+        ...
+    ]
+
+Using the :meth:`WaterBridgeAnalysis.generate_table` method one can reformat
+the results as a flat "normalised" table that is easier to import into a
+database or dataframe for further processing.
+:meth:`WaterBridgeAnalysis.save_table` saves the table to a pickled file. The
+table itself is a :class:`numpy.recarray`.
+
+Detection of water bridges
+---------------------------
+Water bridges are recorded if a bridging water simultaneously form two hydrogen bonds with selection 1 and selection 2.
+
+Hydrogen bonds are detected as is described in \ :class:`~MDAnalysis.analysis.hbonds.hbond_analysis.HydrogenBondAnalysis`, see \ :ref:`Detection-of-hydrogen-bonds`.
+
+The lists of donor and acceptor names can be extended by providing lists of
+atom names in the `donors` and `acceptors` keywords to
+:class:`WaterBridgeAnalysis`. If the lists are entirely inappropriate
+(e.g. when analysing simulations done with a force field that uses very
+different atom names) then one should either use the value "other" for `forcefield`
+to set no default values, or derive a new class and set the default list oneself::
+
+ class WaterBridgeAnalysis_OtherFF(WaterBridgeAnalysis):
+       DEFAULT_DONORS = {"OtherFF": tuple(set([...]))}
+       DEFAULT_ACCEPTORS = {"OtherFF": tuple(set([...]))}
+
+Then simply use the new class instead of the parent class and call it with
+`forcefield` = "OtherFF". Please also consider to contribute the list of heavy
+atom names to MDAnalysis.
+
+How to perform WaterBridgeAnalysis
+-----------------------------------
+
+All water bridges between arginine and aspartic acid can be analysed with ::
+
+  import MDAnalysis
+  import MDAnalysis.analysis.hbonds
+
+  u = MDAnalysis.Universe('topology', 'trajectory')
+  w = MDAnalysis.analysis.hbonds.WaterBridgeAnalysis(u, 'resname ARG', 'resname ASP')
+  w.run()
+
+The results are stored as the attribute
+:attr:`WaterBridgeAnalysis.timeseries`; see :ref:`Analysis Output` for the
+format.
+
+An example of using the :attr:`~WaterBridgeAnalysis.timeseries` would be detecting the precentage of time a certain water bridge exits. ::
+
+  w.timeseries
+  #[ # frame 1
```

Comment:
Don't include output in a code block. Stop the code block, create a new preformatted block (check the sphinx docs), show the output, the continue with the code. It's confusing in an example if you show something (commented out lines) that will not appear in actual output.

Don't cut corners here but describe and document the things that users need to understand.

Reply:
@orbeckst Sorry, I have separated the code block. I have read the sphinx docs and it seems to me that a code block is more suitable to display the result as each character is of fixed width.
Would you mind elaborate a bit about the cut the corner? I thought that having the simplest example will probably make the explanation easier.

Reply:
With "[cutting corners](https://idioms.thefreedictionary.com/cut+corners)" I just meant that you should spend a bit more time on explaining instead of just dumping the output format together with the code. But you have done this in your updates, so that's all looking good to me.

I agree that you need a pre-formatted block to properly show the output format, and the standard Python code block should work just fine for it.